imap_processing/cdf/utils.py

```diff
@@ -163,4 +167,4 @@ def write_cdf(dataset: xr.Dataset):
         terminate_on_warning=True,
     )  # Terminate if not ISTP compliant
 
-    return file_path
+    return [file_path]
```

Comment:
This change is why your tests are failing. Why did you change it to a list? It should return a single path.

Reply:
Without the [] the mypy checks would flag it as an 'Any' type being returns instead of a list. Should the return type for the function then be changed? In order to reflect the single path.

Reply:
It should be a single path returned. Can you enforce the type on the creation of the variable? eg `file_path: Path = ...`